ui/components/app/confirmations/alerts/confirm-alert-modal/confirm-alert-modal.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useCallback, useState } from 'react';
```

Comment:
what if we renamed ConfirmAlertModal to ConfirmMultipleAlertModal? The thought behind this is that is does also handle multiple alerts so this would provide more context